mfr/extensions/codepygments/render.py

```diff
@@ -49,10 +49,11 @@ def _render_html(self, fp, ext, *args, **kwargs):
 
         if exception is not None:
             encoding = chardet.detect(data)
-            try:
-                content = data.decode(encoding['encoding'])
-            except UnicodeDecodeError as e:
-                exception = e
+            if encoding['encoding']:
+                try:
+                    content = data.decode(encoding['encoding'])
+                except UnicodeDecodeError as e:
+                    exception = e
 
```

Comment:
Should an error be raised here `if not encoding['encoding']`? 

I know `AssertionError('Got no content or exception')` will probably be raised, but that's not exactly indicative of what the actual error was.